### PR TITLE
Add NoExecute toleration to rke2Etcd and remove key

### PR DESCRIPTION
**Pull request by aiyengar2**
No other PushProx exporter keys the tolerations, so rke2Etcd's default is being modified for consistency.

Workaround for users who already have Monitoring V2 deployed would be to just override the value in the values.yaml.

If a user needs more fine-tuned tolerations, they can always choose to override that as well.

Related Issue: https://github.com/rancher/rancher/issues/32245

### CODE REVIEW

File: packages/rancher-monitoring/generated-changes/patch/values.yaml.patch [hunk 1]
```diff
@@ -225,8 +225,9 @@
 +    nodeSelector:
 +      node-role.kubernetes.io/etcd: "true"
 +    tolerations:
++      - effect: "NoExecute"
++        operator: "Exists"
 +      - effect: "NoSchedule"
-+        key: node-role.kubernetes.io/master
 +        operator: "Equal"
```
Comment (dramich) [inline packages/rancher-monitoring/generated-changes/patch/values.yaml.patch:231]: Should this get updated to `exists`?

Comment (aiyengar2) [reply packages/rancher-monitoring/generated-changes/patch/values.yaml.patch:231]: yes. Updating

**Review by dramich — DISMISSED** [review]

**Review by aiyengar2 — COMMENTED** [review]
This PR needs to be moved to the dev-v2.5 branch as the dev-v2.5-source branch is now deprecated.

The easiest way to do this would be to create a new local branch from the `dev-v2.5` branch, cherry-pick your commits from the branch this PR points to to that new branch, and then run `make charts` to generate the charts/ and assets/ as part of the new provisioning workflow.

More details on the new workflow can be found in https://github.com/rancher/charts/blob/dev-v2.5/packages/README.md#common-workflow. Feel free to find me on Slack / RocketChat or email me at arvind.iyengar@suse.com for any questions.

**Comment by aiyengar2** [comment]
Rebased to dev-v2.5

**Review by dramich — APPROVED** [review]

**Review by brendarearden — APPROVED** [review]

**Pull request merged**